src/components/Privacy/validateSetConsentOptions.js

```diff
@@ -38,7 +39,20 @@ export default objectOf({
     )
   )
     .nonEmpty()
-    .required()
+    .required(),
+  identityMap: mapOfValues(
+    arrayOf(
+      objectOf({
+        authenticatedState: string(),
+        id: string(),
+        namespace: objectOf({
+          code: string()
+        }).noUnknownFields(),
+        primary: boolean(),
+        xid: string()
```

Comment:
What is `xid`?

Reply:
https://github.com/adobe/xdm/blob/master/schemas/context/identity.schema.json